website/js/index.js

```diff
@@ -0,0 +1,20 @@
+const fileInput = document.getElementById('file-input');
+document.getElementsByTagName("h1")[0].innerHTML = "main";
+fileInput.addEventListener('change', (e) => displayImage(e.target.files));
+
+function displayImage(fileList) {
+  document.getElementsByTagName("h1")[0].innerHTML = "attempting to display";
```

Comment:
Recommendation: use https://getbootstrap.com/docs/4.3/components/spinners/ since we're already using Bootstrap. Should be reasonably quick to copy in.

PS: You'll want to set the style property to "display: none" _just before_ you update the image preview: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/style

Reply:
Note: this text isn't update when the file is changed.